lib/ansible/module_utils/openstack.py

```diff
@@ -114,15 +114,20 @@ def openstack_cloud_from_module(module, min_version='0.12.0'):
         # Due to the name shadowing we should import other way
         import importlib
         sdk = importlib.import_module('openstack')
+        sdk_version = importlib.import_module('openstack.version')
     except ImportError:
         module.fail_json(msg='openstacksdk is required for this module')
 
     if min_version:
```

Comment:
Maybe we can remove the `if min_version:`? min_version has a useful default, and if `min_version=None` is passed, it is OK to explode, from my point of view.

Reply:
It's not OK ([see](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html#handling-exceptions-bugs-gracefully)).